testcontainers/tests/async_runner.rs

```diff
@@ -81,6 +81,11 @@ async fn start_containers_in_parallel() -> anyhow::Result<()> {
 
     let image = GenericImage::new("hello-world", "latest").with_wait_for(WaitFor::seconds(2));
 
+    // Make sure the image is already pulled, since otherwise pulling it may cause the deadline
+    // below to be exceeded.
+    let run_pull = image.clone().start().await?;
+    drop(run_pull);
```

Comment:
Actually, there is more straightforward way to pull the image:
https://docs.rs/testcontainers/0.23.1/testcontainers/runners/trait.AsyncRunner.html#tymethod.pull_image

Reply:
Makes sense! Done.